### PR TITLE
fix: Schema Location for findbugsfilter

### PR DESCRIPTION
**Pull request by NeetigyaPod**
Before opening a 'pull request'

* Search existing issues and pull requests to see if the issue was already discussed.
* Check our discussions to see if the issue was already discussed.
* Check for specific project we support to raise the issue on, under [spotbugs](https://github.com/spotbugs)
* Do not open intellij plugin issues here, open them at [intellij-plugin](https://github.com/JetBrains/spotbugs-intellij-plugin) *

----

Make sure these boxes are checked before submitting your PR -- thank you!

- [x] Added an entry into `CHANGELOG.md` if you have changed SpotBugs code

Closes: #1416 . Added xsi:schemaLocation to solve issue.

### CODE REVIEW

**Review by ThrawnCA — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 - Do not report SIC_INNER_SHOULD_BE_STATIC for classes annotated with JUnit Nested ([#560](https://github.com/spotbugs/spotbugs/issues/560))
 - Detect created, but not-thrown exceptions, which are created by not the constructor ([#2547](https://github.com/spotbugs/spotbugs/issues/2547))
 - Fixed eclipse plugin Effort.values pass to effortViewer as required cast to varargs ([#2579](https://github.com/spotbugs/spotbugs/pull/2579))
-
+- Fixed schema location for findbugsfilter.xsd ([[#1416](https://github.com/spotbugs/spotbugs/issues/1416)])
```
Comment (hazendaz) [inline CHANGELOG.md:43]: @NeetigyaPod This needs moved into new section at top as release just occurred, make sure you are rebased and you an create similar section as you used here in the unreleased area at top.

File: spotbugs/etc/findbugsfilter.xsd [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <schema targetNamespace="https://github.com/spotbugs/filter/3.0.0" elementFormDefault="unqualified"
```
Comment (hazendaz) [inline spotbugs/etc/findbugsfilter.xsd:2]: @NeetigyaPod YOu need the other side of this that I has put on that ticket.  This itself is broken.  The schema location is a list of property=value pairs where the left side is the namespace (what you have now) and the right is the location of the actual xsd.  Note that the namespace is not actually an endpoint.  So the purpose is so you can link to the source right from your IDE.

Comment (NeetigyaPod) [reply spotbugs/etc/findbugsfilter.xsd:2]: Oh makes sense. Committing the change and pushing, thank you.

File: CHANGELOG.md (hunk 1 above)
Comment (NeetigyaPod) [reply CHANGELOG.md]: Rectified.

**Pull request merged**